src/lib/motors/SmartServo.cpp

```diff
@@ -235,7 +235,18 @@ float SmartServoClass::getPosition(uint8_t const id)
     return -1.0f;
 
   mbed::ScopedLock<rtos::Mutex> lock(_mtx);
-  return positionToAngle(readWordCmd(id, REG(SmartServoRegister::POSITION_H)));
+  int position = readWordCmd(id, REG(SmartServoRegister::POSITION_H));
```

Comment:
```suggestion
  int const position = readWordCmd(id, REG(SmartServoRegister::POSITION_H));
```

Reply:
Thanks, @aentinger! I have committed the suggested change.